pkgs/applications/editors/ne/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, ncurses, texinfo, tetex, perl, ghostscript }:
+
+
+stdenv.mkDerivation {
+  name = "ne-3.0.1";
+  version = "3.0.1";
+  src = fetchurl {
+    url = https://github.com/vigna/ne/archive/3.0.1.tar.gz;
+    md5 = "ec3dc82d10c271e95ecde630538f9f19";
+  };
+  buildInputs = [ ncurses tetex texinfo perl ghostscript ];
+  preBuild = ''
+    substituteInPlace src/makefile --replace "CC=c99" "cc=gcc"
+    substituteInPlace src/makefile --replace "-lcurses" "-lncurses"
+    substituteInPlace makefile --replace "./version.pl" "perl version.pl"
+    cd doc && make && cd ..
```

Comment:
Is there not a root `Makefile` that would allow this to work without you specifying it? The standard `buildPhase` will run `make` if you leave it alone.